mkdocs.yml

```diff
@@ -71,6 +71,9 @@ nav:
       - Guidelines: develop/contributing.md
       - Setup: develop/setup.md
       - Testing: develop/testing.md
+  - Latest Updates:
```

Comment:
Suggesting this to be the 2nd tab after "What is AAQ?" and title it `⚠️ Latest Updates` or `📢 Latest Updates` instead.
Other emoji option - 🚨❗

Reply:
Great! Moved it.